fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2881,9 +2881,14 @@ public class Config extends ConfigBase {
     public static boolean warehouse_idle_check_enable = false;
 
     // e.g. "tableId1;tableId2"
-    @ConfField(mutable = true)
+    @ConfField(mutable = true, comment = "Deprecated")
```

Comment:
how about using `@ConfField(mutable = true, aliases = {"lake_compaction_disable_ids"})`

Reply:
i put this aliases to lake_compaction_disable_tables

Reply:
i put this aliases to lake_compaction_disable_tables, so we can just use lake_compaction_disable_ids